### ISSUE TITLE
REQ: More objects in library

### ISSUE THREAD

**Issue by bmix** [fwbuilder/fwbuilder#16]
I remember, that even back in the day, there were no user-libraries being published on the web. Now, usually, one creates a library with services they use and then backs them up for later, but this is theory. I happened to kill both my dev-machine's disk *and* the backup! And now I am at start again.

So, therefore my request: Could additional libraries being published here? Let's say, as a seperate prject, next to 'fwbuilder/fwbuilder'? Like 'fwbuilder/library'?

I am sure, some of you have the code already (otherwise, I'd contribute). Thanks.

**Comment by sirius** [comment]
@bmix Can you elaborate? You share your objects library with other users?

Sure, we can create a fwbuilder/library repo.

**Comment by bmix** [comment]
No, I did not share any objects with other users, but I always wished, there'd be a central repo, since a lot of protocols are not in the library by default. Of course, I am mainly talking about (exotic) protocols, since it may not make much of a sense to share network-objects (hosts, firewalls, networks) with others. I just remember having defined a lot of them by hand and then I had both of my backup disks go bang, and all was lost, when my main disk went bang, too.

On the other side... If I am not being mistaken, then the IANA publishes their *Service Name and Transport Protocol Port Number Registry* file as XML. Lemme check....Yep, they do! Should not be a problem to write an XSL-T, since the fwbuilder library is XML, too. And as it seems...I already started such an XSLT. I completly forgot about it, lol.

Sorry for late reply, I totally missed your answer. I will come back to you with this.
